packages/protocol/contracts/common/Accounts.sol

```diff
@@ -498,7 +464,7 @@ contract Accounts is
    * @return Patch version of the contract.
    */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 1, 4, 2);
+    return (1, 1, 4, 3);
```

Comment:
This will probably need to be `1.2.0.0` once the release check starts working again.